### PR TITLE
Create solution.py

### PR DESCRIPTION
**Pull request by winry-fruitiwi**

### CODE REVIEW

File: 2833. Furthest Point From Origin/solution.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+class Solution:
+    def furthestDistanceFromOrigin(self, moves: str) -> int:
+        # iterate through commands
+            # use switch-case statement to check for each command
```
Comment (winry-fruitiwi) [inline 2833. Furthest Point From Origin/solution.py]: Sorry, this is actually a match-case statement, it's switch-case in JS and Java.

File: 2833. Furthest Point From Origin/solution.py [hunk 2]
```diff
@@ -0,0 +1,30 @@
+class Solution:
+    def furthestDistanceFromOrigin(self, moves: str) -> int:
```
Comment (aeruru) [inline 2833. Furthest Point From Origin/solution.py:5]: No function comment

File: 2833. Furthest Point From Origin/solution.py [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+        # number of _ moves
+        numUnknownMoves = 0
+        currentPos = 0
+        
+        # iterate through commands
+        for command in moves:
+            # use match-case statement to check for each command
+            match command:
+                # case R: increment
```
Comment (aeruru) [inline 2833. Furthest Point From Origin/solution.py]: this is obvious in the code but you don't explain why

File: 2833. Furthest Point From Origin/solution.py [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+                
+                # case L: decrement
+                case "L":
+                    currentPos -= 1
+            
+                # case _: increment number of _ moves
+                case "_":
+                    numUnknownMoves += 1
+        
+        # check sign of current position
```
Comment (aeruru) [inline 2833. Furthest Point From Origin/solution.py]: Hmm this is a very roundabout way of getting the answer. Can you think of a more straightforward way?

**Review by aeruru — APPROVED** [review]

**Pull request merged**